src/commands/commands.h

```diff
@@ -39,5 +39,6 @@ int Graph_Debug(RedisModuleCtx *ctx, RedisModuleString **argv, int argc);
 int Graph_Delete(RedisModuleCtx *ctx, RedisModuleString **argv, int argc);
 int Graph_Config(RedisModuleCtx *ctx, RedisModuleString **argv, int argc);
 int Graph_Slowlog(RedisModuleCtx *ctx, RedisModuleString **argv, int argc);
+GRAPH_Commands CommandFromString(const char *cmd_name);
```

Comment:
Please try and keep the "paragraph" ordered by line length

Reply:
Both are important to me.
There's a general format I'm trying to apply to this project, which I require all members to follow.